### PR TITLE
Release 0.12.0

### PR DESCRIPTION
**Pull request by JohanMabille**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by JohanMabille** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml
Comment (jjerphan) [inline recipe/meta.yaml:1]: Applying this patch and rendering after it resolves all the warnings of `conda-lint` and the packaging issue on my machine:
```diff
diff --git i/recipe/meta.yaml w/recipe/meta.yaml
index c90d539..510a776 100644
--- i/recipe/meta.yaml
+++ w/recipe/meta.yaml
@@ -16,13 +16,14 @@ build:
 
 requirements:
   build:
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - cmake
     - make  # [unix]
   host:
     - nlohmann_json >=3.11.2
   run:
-    - {{ pin_compatible('nlohmann_json', max_pin='x.x.x') }}
+    - {{ pin_compatible('nlohmann_json', max_pin='x.x.x') }}
 
 test:
   commands:
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,9 +20,9 @@ requirements:
     - cmake
     - make  # [unix]
   host:
-    - xtl >=0.7.0,<0.8
+    - nlohmann_json >=3.11.2
   run:
-    - xtl >=0.7.0,<0.8
+    - {{ pin_compatible('nlohmann_json', max_pin='x.x.x') }}
```
Comment (jjerphan) [inline recipe/meta.yaml]:
```suggestion
    - {{ pin_compatible('nlohmann_json', max_pin='x.x.x') }}
```

Comment (JohanMabille) [reply recipe/meta.yaml]: Good catch, thanks a lot!

**Comment by JohanMabille** [comment]
@conda-forge-admin please rerender

**Pull request merged**